### PR TITLE
Transaction details

### PR DESCRIPTION
**Pull request by dylanlott**
Adds collapse panel on each transaction that shows more details for that transaction.

### CODE REVIEW

**Comment by dylanlott** [comment]
I updated this with master, so this has the most current reset password updates in it as well.

File: src/views/Billing/Transaction-List.vue [hunk 1]
```diff
@@ -101,7 +112,7 @@ export default {
           const transaction = {...credit};
           transaction.amount = -credit.paid_amount;
           transaction.type = `${credit.type} payment`;
-          transaction.description = `- Thank you! 🐶`;
```
Comment (aleitner) [inline src/views/Billing/Transaction-List.vue:104]: Put this dog emoji back in or else

**Comment by aleitner** [comment]
Can you change `getMessage` to be `Completed` instead of `Paid`. Payment processors usually mark paid when processing

File: src/views/Billing/Transaction-List.vue (hunk 1 above)
Comment (aleitner) [reply src/views/Billing/Transaction-List.vue:104]: https://blog.farrant.me/adding-emoji-support-to-any-website/

**Comment by RichardLitt** [comment]
👋 Hey! Thanks for this contribution. Apologies for the delay in responding!

We've decided to rearchitect Storj, so that we can scale better. You can read more about this decision [here](https://storj.io/blog/2018/01/getting-from-petabytes-to-exabytes-the-road-ahead/). This means that we are entirely focused on v3 at the moment, in the [storj/storj](https://github.com/storj/storj) repository. Our white paper for v3 is coming **very, very** soon - follow along on [the blog](https://storj.io/blog) and in our [Rocketchat](https://community.storj.io/home).

As this repository is part of the v2 network, we're no longer maintaining this repository. I am going to close this for now. If you have any questions, I encourage you to jump on Rocketchat and ask them there. Thanks!

**Pull request closed without merging**